CHANGELOG.md

```diff
@@ -1,4 +1,57 @@
 # Changelog - v3
+## [v3.6.0] (June 28 2023)
+
+### Feat:
+* Official support for Feature Configuration
+  - You can now configure the features of UIKit through the `uikitOptions` prop of `<SendbirdProvider />` or `<App />` component. You can also find the detailed sample usage from [SAMPLE.md#UIKit-Configuration-Samples](./SAMPLES.md#UIKit-Configuration-Samples)
+  - The minimum `@sendbird/chat` version has been increased to 4.9.2.
+```jsx
+  <SendbirdUIKitContainer
```

Comment:
Does this mean SendbirdProvider ?

Reply:
Oh right. Thanks for catching it out. https://github.com/sendbird/sendbird-uikit-react/pull/662/commits/a5cb0184590f0f2075e41d4c3b3c3977f1dc3c95 Fixed in here.